flake.nix

```diff
@@ -90,6 +90,16 @@
         inherit pkgs;
       };
     };
+    apps.${system} = {
+      # Run the VM with `nixos run .#vm`
+      vm = {
```

Comment:
There is also:

```
$ nixos-shell --flake .#vm
```

which also mounts the user home and shell into the vm.

Reply:
oops, das hatte ich übersehen.

@Mic92 @astro besprecht ihr miteinander welche lösung ihr bevorzugt?